operations/enterprises.md

```diff
@@ -628,6 +628,57 @@ Adds a new company to the enterprise.
 | `AccountingCode` | string | optional | Accounting code of the company. |
 | `Address` | [Address](configuration.md#address) | optional | Address of the company. |
 
+### Response
+
+Same structure as in [Get all companies](enterprises.md#get-all-companies) operation.
+
+## Update company
+
+Updates information about a company. Note that if any of the fields are sent as `null`, it won't clear the field in Mews. If the `value` within the object is sent as `null`, the field will be cleared in Mews.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/companies/update`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "CompanyId": "9ebf6446-64c3-4f85-943c-aa540092b963",
+    "Name": {
+        "Value": "Golden Tech Company"
+    },
+    "MotherCompanyId": null,
+    "Identifier": null,
+    "TaxIdentifier": null,
+    "AdditionalTaxIdentifier": null,
+    "BillingCode": null,
+    "AccountingCode": {
+        "Value": "1234"
+    }
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `CompanyId` | string | required | Unique identifier of the [Company](enterprises.md#company). |
+| `Name` | String [Value](enterprises.md#update-value) | optional | Name of the company \(if it is non-empty, otherwise `null`\). |
+| `MotherCompanyId` | String [Value](enterprises.md#update-value) | optional | Unique identifier of the mother company \(if it is non-empty, otherwise `null`\). |
+| `Identifier` | String [Value](enterprises.md#update-value) | optional | Identifier of the company (e.g. legal identifier) \(if it is non-empty, otherwise `null`\). |
+| `TaxIdentifier` | String [Value](enterprises.md#update-value) | optional | Tax identification number of the company \(if it is non-empty, otherwise `null`\). |
+| `AdditionalTaxIdentifier` | String [Value](enterprises.md#update-value) | optional | Additional tax identifer of the company \(if it is non-empty, otherwise `null`\). |
+| `BillingCode` | String [Value](enterprises.md#update-value) | optional | Billing code of the company \(if it is non-empty, otherwise `null`\). |
+| `AccountingCode` | String [Value](enterprises.md#update-value) | optional | Accounting code of the company \(if it is non-empty, otherwise `null`\). |
+
+#### Update value
```

Comment:
Actually, the simplest and most straightforward way, how to desrcibe it here in the docs is to just put here `String Update Value`, `Number Update Value`, `Bool Update Value` etc. for all types instead of introducing "generics" like this. It is little more redundant, but much more understandable...